scipy/stats/_qmc.py

```diff
@@ -914,13 +910,48 @@ class LatinHypercube(QMCEngine):
 
         .. versionadded:: 1.8.0
 
+    strength : {1, 2}, optional
+        Strength of the LHS. ``strength=1`` produces a plain LHS while
+        ``strength=2`` produces an orthogonal array based LHS of strength 2
+        [7]_, [8]_. In that case, only ``n=p**2`` points can be sampled,
+        with ``p`` a prime number. It also constrains ``d <= p + 1``.
+        Default is 1.
+
+        .. versionadded:: 1.8.0
+
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+
+    When LHS is used for integrating a function :math:`f` over :math:`n`,
+    LHS is extremely effective on integrands that are nearly additive [2]_.
+    With a LHS of :math:`n` points, the variance of the integral is always
+    lower than plain MC on :math:`n-1` points [3]_. There is a central limit
+    theorem for LHS on the mean and variance of the integral [4]_, but not
+    necessarily for optimized LHS due to the randomization.
+
+    :math:`A` is called an orthogonal array of strength :math:`t` if in each
+    n-row-by-t-column submatrix of :math:`A`: all :math:`p^t` possible
+    distinct rows occur the same number of times,
+    with :math:`p` a prime number. The elements of :math:`A` are in the set
+    :math:`\{0, 1, ..., p\}`, also called symbols.
+
+    To create an orthogonal array based LHS, the array is randomized by
+    randomly permuting the symbol values on each column independently. Then,
+    on each column and for a given symbol value, a plain LHS is added. The
```

Comment:
How do you add a `n x d` plain LHS "on each column for a given symbol value"?

Maybe this is too difficult to describe, and describing how to get from a LHS to an orthogonal array is easier.

"A Latin hypercube `A` is said to be orthogonal of strength 1 if...
<formula>
is an orthogonal array of strength 1.

A Latin hypercube `A` is said to be orthogonal with strength 2 if...
<formula>
is an orthogonal array of strength 2.
"

Reply:
> Presumably "orthogonal array based" LHS means "strength 2" LHS, even though both of them can be thought of as relating to orthogonal arrays?

Yes.

> I suppose "plain LHS" is "strength 1" LHS?

Yes. It's the classical/original formulation.

> How do you add a `n x d` plain LHS "on each column for a given symbol value"?

It's actually a `px1` plain LHS which is used. Ex, for a given column, all 0 would be replaced by a this LHS. You have to construct p LHS per column.

> Maybe this is too difficult to describe, and describing how to get from a LHS to an orthogonal array is easier.
> 
> "A Latin hypercube `A` is said to be orthogonal of strength 1 if...
> (formula)
> is an orthogonal array of strength 1.
> 
> A Latin hypercube `A` is said to be orthogonal with strength 2 if...
> (formula)
> is an orthogonal array of strength 2.
> "
> 
> But I suppose if I have concerns about the documentation I can change them myself after this merges.

No sure about adding formulas here. I mean at most in the literature they just define something like `OA(d, n, t, l)`. IMO this would not help more than a plain explanation (which hopefully we should be able to write).

I will have another read at this.